nat-lab/tests/telio.py

```diff
@@ -302,7 +304,7 @@ def _handle_derp_event(self, line) -> bool:
         result = re.search("{(.*)}", json_string)
         if result:
             derp_server_json = DerpServer.schema().loads(
-                "{" + result.group(1).replace("\\", "") + "}"
+                "{" + result.group(1).replace("\\", "") + ',"used":false' + "}"
```

Comment:
when running 3.6, won't this be duplicated? `"used":false,"used":false`

Reply:
It should, but do not see a problem, test only fail when there is an unknown variable, or lack of variable.

Can add a comment that  this is a workaround.

Reply:
probably issue is hidden somewhere, can you atleast add TODO and create Jira issue to investigate this, if you are not willing to make this properly :smile: or maybe add `used: Optional[bool] = None` to class itself, this might solve the issue :thinking: